templates/sag-cc-mirrors/README.md

```diff
@@ -18,10 +18,12 @@
 -->
 # Command Central Remote Mirrors Registration
 
-Use this template to register product and fix mirror repositories in Command Central.
+Use this template to add remote mirror repositories for products and fixes. This template connects Command Central to the mirror repositories that already exist on a remote Platform Manager node.
 
 ## Requirements
 
+None.
```

Comment:
The user needs to create DEFAULT_ADMINISTRATOR credentials for SPM that hosts the mirrors (it can be done by sag-cc-creds)

Reply:
Updated.